Microsoft.Recognizers.Text.DateTime/English/Utilities/EnlighDatetimeUtilityConfiguration.cs

```diff
@@ -21,28 +21,21 @@ public class EnlighDatetimeUtilityConfiguration : IDateTimeUtilityConfiguration
             "in",
         };
 
-        public static readonly List<string> AmStringList = new List<string>
-        {
-            "am",
-            "a.m.",
-            "a",
-        };
+        public static readonly string AmPrefix = "a";
+        public static readonly string PmPrefix = "p";
+        public static readonly string AmPmPrefix = "ampm";
 
-        public static readonly List<string> PmStringList = new List<string>

```

Comment:
the DescRegex defined that only am/pm related strings will be matched. "at" will be filtered by DescRegex in extractor.
def: (?<desc>pm\b|am\b|p\.m\.|a\.m\.|p\b|a m\b|p m\b|a. m.\b|p. m.\b|ampm)